### PR TITLE
chore(pi-strings): remove deadcode

### PR DESCRIPTION
**Pull request by karlek**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -189,7 +189,7 @@ jobs:
 
   build-pi-strings:
     docker:
-      - image: circleci/rust:1.38
+      - image: karlekdocker/pippi:0.0.6
```
Comment (mewmew) [inline .circleci/config.yml:192]: We have our own image now :) How does karlekdocker/pippi differ from the default Rust image of CircleCI?

Comment (mewmew) [reply .circleci/config.yml:192]: Oh, is it the addition of `bash`?

Comment (karlek) [reply .circleci/config.yml:192]: For instance it has `cmake` and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile

Comment (karlek) [reply .circleci/config.yml:192]: > Oh, is it the addition of `bash`?

No, both circleci/rust and karlekdocker/pippi has bash

Comment (mewmew) [reply .circleci/config.yml:192]: > For instance it has cmake and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile

Thanks! Mayhaps you could give me the Docker one0one crash course later :)

Comment (mewmew) [reply .circleci/config.yml:192]: Actually, taking a look at the `Dockerfile` it is very clean and only contains things that are logical requirements of our build pipeline.

**Edit:** except perhaps Perl :p

Comment (karlek) [reply .circleci/config.yml:192]: > > For instance it has cmake and all protobuf related tools. You can see here: https://github.com/lapsang-boys/pippi/blob/master/.circleci/images/primary/Dockerfile
> 
> Thanks! Mayhaps you could give me the Docker one0one crash course later :)

Fo shizle

**Pull request merged**